packages/components/field-label/package.json

```diff
@@ -22,6 +22,7 @@
     "@babel/runtime": "^7.19.0",
     "@babel/runtime-corejs3": "^7.19.1",
     "@commercetools-uikit/constraints": "15.5.1",
+    "@commercetools-uikit/design-system": "15.5.1",
     "@commercetools-uikit/icon-button": "15.5.1",
     "@commercetools-uikit/icons": "15.5.1",
     "@commercetools-uikit/label": "15.5.1",
```

Comment:
I think you need to rebase your branch, as the latest published version is `15.6.0`